packages/core/package.json

```diff
@@ -23,6 +23,7 @@
     "build": "tsc -b ."
   },
   "dependencies": {
+    "@finnair/path": "0.4.0",
```

Comment:
Why does core depend of path? Shouldnt be the other way aroung

Reply:
Core is v-validation (core) and it's just one thing using `path`. Also, I'm not sure anymore why we decided to name it `v-validation-core` instead of just `v-validation` 🤔

Reply:
@paazmaya @Jsemi Do you think we could rename `v-validation-core` to `v-validation`?